content/en/careers/le-sr-sw-eng.html

```diff
@@ -9,7 +9,11 @@
 <section class="slice slice-sm">
 <div class="container">
 
-<p>Posted: <strong>June 18, 2020</strong><br>Position Status: <strong>Open</strong></p>
+<p>
+Posted: <strong>June 18, 2020</strong><br>
+Position Status: <strong>Open</strong><br>
+Location: <strong>100% Remote within US and Canada</strong><br>
```

Comment:
Maybe we get rid of the 100% part? I'm not sure saying that makes more sense than just saying the job is remote.